dbt/adapters/dremio/api/parameters.py

```diff
@@ -42,20 +42,20 @@ class ParametersBuilder:
     @classmethod
     def build(cls, credentials: DremioCredentials):
         if (
-            credentials.cloud_host != None
-            and credentials.software_host == None
-            and credentials.cloud_project_id != None
+            credentials.cloud_host is not None
```

Comment:
This is a great catch. We should always use is/is not for None and ==/!= for everything else.

Reply:
That's not me - that is flake8!! :P